src/app/legacy/components/FrostedGlassPromo/TimestampFooter.jsx

```diff
@@ -12,7 +12,8 @@ import PromoTimestamp from '#components/Promo/timestamp';
 
 const StyledTimestamp = styled(PromoTimestamp)`
   ${({ service }) => service && getSansRegular(service)}
-  color: ${({ isAmp }) => (isAmp ? 'black' : '#e6e8ea')};
+  color: ${props =>
+    props.isAmp ? props.theme.palette.BLACK : props.theme.palette.GREY_3};
```

Comment:
Is it better to do this ^ or

```
color: ${({ isAmp }) =>
    props =>
      isAmp ? props.theme.palette.BLACK : props.theme.palette.GREY_3};
```

Reply:
Your change looks fine to me. The other example you give could be even shorter:

```jsx
color: ${({ isAmp, theme }) => isAmp ? theme.palette.BLACK : theme.palette.GREY_3};
```

But I think it's good to be explicit about `props`, the example above kind of obfuscates where those values are coming from if you don't have prior knowledge about emotions styled API

Reply:
Cool, thanks for clarifying!